.eslintrc

```diff
@@ -23,6 +23,7 @@
     "react/prop-types": "off",
     // should be turned on eventually, but not compatible with current codebase
     "no-shadow": "off",
+    "no-console":"off",
```

Comment:
I'm fine with this being a warn but not an error? but we can also leave it off for now

Reply:
Changed to warn